### PR TITLE
Added theory resources for lesson 6

### PR DESCRIPTION
**Pull request by MartenBE**

### CODE REVIEW

**Review by thomasaelbrecht — CHANGES_REQUESTED** [review]
Maybe we should integrate what we had on the learning path in Chamilo?

File: docs/06-monitoring.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+
+### Online courses
+
+-   DevOps Foundations: Monitoring and Observability, Linkedin Learning:
+https://www.linkedin.com/learning/devops-foundations-monitoring-and-observability/
+    - Students of HOGENT get free access to LinkedIn Learning through [Academic Software](https://academicsoftware.eu/login).
+
+### Books
+
+-   https://link.springer.com/search?query=monitoring&facet-content-type=%22Book%22&facet-discipline=%22Computer+Science%22
```
Comment (thomasaelbrecht) [inline docs/06-monitoring.md]: This is from Chamilo, maybe we should add one or two specific books? I have one from Oreilly (https://www.oreilly.com/library/view/prometheus-up/9781492034131/), but it's not free.

**Review by thomasaelbrecht — CHANGES_REQUESTED** [review]

File: docs/06-monitoring.md (hunk 1 above)
Comment (MartenBE) [reply docs/06-monitoring.md]: I think it is better to mention no books, than to offer paid books. There is a lot of info on prometheus etc. online already.

Comment (thomasaelbrecht) [reply docs/06-monitoring.md]: Yes indeed, I bought this one in a bundle of 15 books for $15. I'm not aware of any good books for monitoring. Remove this section! 😉

**Review by thomasaelbrecht — APPROVED** [review]

**Pull request merged**